campfire/ui/src/components/MediaInput.tsx

```diff
@@ -60,6 +69,25 @@ export function MediaInput() {
           ))}
         </select>
       </div>
+      <div className="AudioOutputs w-full">
+        <Text variant="label" pb={1}>
+          Audio Output
+        </Text>
+        <select
+          className="input default-ring bg-gray-200"
+          onChange={onAudioOutChange}
+          defaultValue={audioOutputDevices.findIndex(d => {
+            return d.deviceId === mediaStore.outputSoundDevice.deviceId;
+          })}
+          disabled={!('sinkId' in HTMLMediaElement.prototype)}
```

Comment:
helpful line to disable this selector in browsers that dont support setting sinkId. Taken from [WebRTC example](https://github.com/webrtc/samples/blob/gh-pages/src/content/devices/input-output/js/main.js#L17)

Maybe could switch this to `'setSinkId'` but I'm just folowing the example. Probably doesn't matter